### DISCUSSION TITLE
LISEM landslide runout model

### DISCUSSION THREAD

**Discussion by lozaayehu** [bastianvandenbout/LISEM#10] [Q&A]
How can we determine the initial boudary condition of (initial water height and initial solid height maps)? How can we calculate that using a DEM map?

**Comment by bastianvandenbout** [comment]
Hi, 

So there is a couple options here:
1) Slope stability modelling. There are some slope stability models that you can run to determine potential failure areas and their initial heights. So run something like SlopeStabilityIS/SlopeStabilityIFM to do that. You will need good data on soil depth/cohesion/internal friction angle, or calibrate for that based on a mapped landslide impact extent. Once you know failure, you can estimate soildepth * porosity = water height, and soildepth * (1.0-porosity) = solid height. Typically, failures occur during saturation, but otherwise compensate for that
2) Use mapped extent and assume the soil depth fails for this area. This requires soil depth info (combining physically based soil depth modelling with observaitons perhaps). Some example script for soil depth estimation is:
void main() 
{ 

    af.map = Accuflux2D2(dem.map,dem.map * 0.0 + 0.01,dem.map * 0.0 + 0.01,dem.map * 0.0,dem.map * 0.0,300)[0]; 

    soildepth.map = AccufluxSoil(dem.map,1000.0*af.map,dem.map *0.0 +2.0,0.001,2.0,100,0.1); 

    soildepth2.map = SteadyStateSoil(dem.map,dem.map * 0.0 + 0.1,dem.map * 0.0 + 0.1,100); 

} 
3) Finally, there are a lot of relationships for area to volume of landslides. If you have the initial extent, you can distribute the estimated vollume from such a relationship over the extent and estimate height. See Table 1 from: Guzzetti, F., Ardizzone, F., Cardinali, M., Rossi, M., & Valigi, D. (2009). Landslide volumes and landslide mobilization rates in Umbria, central Italy. Earth and Planetary Science Letters, 279(3-4), 222-229. 

Hope this helps, let me know!

**Comment by lozaayehu** [comment]
Dear Professor,
Thank you very much for your message. I am working on developing the soil depth map and then, I will use the approach for slope stability analysis.
Best regards.
Loza
________________________________
